docs/osmosis-core/relaying/relayer-guide.md

```diff
@@ -191,13 +191,13 @@ store_prefix = 'ibc'
 default_gas = 5000000
 max_gas = 15000000
 gas_price = { price = 0.0026, denom = 'uosmo' }
-gas_adjustment = 0.1
+gas_multiplier = 1.1
 max_msg_num = 20
 max_tx_size = 209715
 clock_drift = '20s'
 max_block_time = '10s'
 trusting_period = '10days'
-memo_prefix = 'Relayed by Czar'
+memo_prefix = 'Osmosis Docs Rocks'
```

Comment:
This is an arbitrary string that Hermes will attach to each transaction. Each relayer operator can put here their own brand here (example: see the Memo field [here](https://www.mintscan.io/regen/txs/7AFB7F451DB183BE1EB0856C9F3EF19E521F6D389411C5A583CFDA2B0685A1BE)).